clientapp/app/components/EnterMeasurementsPage/EnterMeasurements.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Measurements from '../shared/Measurements'
+
+export default function EnterMeasurements() {
+  return (
+    <div>
+      <Measurements/>
```

Comment:
Any reason why this Measurements component was made a reusable component? Are you planning on using this Measurements component in other pages?